.changeset/popular-gifts-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
```

Comment:
Should this be a patch as it seems to fix an issue?
```suggestion
"@kaizen/components": patch
```

Reply:
Yep, thanks